lib/src/layer/marker_layer.dart

```diff
@@ -191,10 +196,30 @@ class _MarkerLayerState extends State<MarkerLayer> {
   var _pxCache = <CustomPoint>[];
 
   // Calling this every time markerOpts change should guarantee proper length
```

Comment:
Should this be a documentation comment (use '///')?

```suggestion
  /// Calling this every time markerOpts change should guarantee proper length
```